src/Article/votes.jsx

```diff
@@ -1,6 +1,33 @@
-function Votes() {
-    return (<>
-    these are the votes</>)
+import { useState } from "react"
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
+import { patchArticle } from "../../Utils/patchArticles";
+
+function Votes({ setArticle, article }) {
+
+
+    function handleUpClick() {
+        setArticle((currArticle) => {
+          return {...currArticle, votes:currArticle.votes + 1}
+        })
```

Comment:
Could we extract the votes out of the article into state so we don't have to update the whole article state?